vllm/model_executor/layers/attention/attention.py

```diff
@@ -34,7 +50,7 @@ def __init__(
         sliding_window: Optional[int] = None,
     ) -> None:
         super().__init__()
-        if _use_flash_attn():
+        if False and _use_flash_attn():
```

Comment:
nit: let's add a todo?

Reply:
this is TODO before merging this PR! Please review it without this first